pkg/config/config.go

```diff
@@ -37,7 +37,8 @@ type Asset struct {
 
 // Phoenix defines the available phoenic configuration.
 type Phoenix struct {
-	Path string
+	Path      string
+	Namespace string
```

Comment:
This should be part of HTTP and not Phoenix, as this will be done for all the services.

Reply:
But namespace provides discovery whether it is grpc or http. I'd rather put it under `Config`, like so:

```go
type Config struct {
	File    	string
	Namespace 	string
	Log     	Log
	Debug   	Debug
	HTTP    	HTTP
	Tracing 	Tracing
	Asset   	Asset
	Phoenix 	Phoenix
}
```

Will open a PR